tests/wallet/test_wallet_user_store.py

```diff
@@ -0,0 +1,57 @@
+import asyncio
+from pathlib import Path
+import aiosqlite
+import pytest
+
+from chia.util.db_wrapper import DBWrapper
+from chia.wallet.util.wallet_types import WalletType
+
+from chia.wallet.wallet_user_store import WalletUserStore
+
+
+@pytest.fixture(scope="module")
+def event_loop():
+    loop = asyncio.get_event_loop()
+    yield loop
+
+
+class TestWalletUserStore:
```

Comment:
This class could be dropped which would save an indent for the whole test.

Reply:
Why canwe drop this?

Reply:
The class just isn't used in any way, is it? You can just have the tests in there, see https://github.com/Chia-Network/chia-blockchain/blob/main/tests/farmer_harvester/test_farmer_harvester.py.